test/e2e/framework/util.go

```diff
@@ -2038,6 +2038,11 @@ func RandomSuffix() string {
 	return strconv.Itoa(r.Int() % 10000)
 }
 
+// ExpectError expects an error happens, otherwise an exception raises
+func ExpectError(err error, explain ...interface{}) {
```

Comment:
I'm genuinely surprised this has not existed for a while.

Reply:
Hehe, yeah right. Other guys also said the same thing.
Thanks for your approval anyways :-)